glossary.asciidoc

```diff
@@ -92,63 +92,61 @@ Breach Remedy Transaction::
      At any point, the channel can be closed by either Alice or Bob if they submit their commitment transaction to the Bitcoin blockchain.
-     Submitting an older (outdated) commitment transaction is considered "cheating" (i.e. protocol breach) in the Lightning network and can be penalized by the other party claiming all the funds in the channel for themselves.     
+     Submitting an older (outdated) commitment transaction is considered "cheating" (i.e. a protocol breach) in the Lightning network and can be penalized by the other party, claiming all the funds in the channel for themselves.     
 
 computationally easy::
-    A problem is considered to be computationally easy if there exists an algorithm that is able to compute the solution to the problem rather quickly (in polynomial time complexity). 
+    A problem is considered to be computationally easy if there exists an algorithm that is able to compute the solution to the problem relatively quickly (in polynomial time complexity). 
 
 computationally hard::
-    A problem is considered to be computationally hard if no algorithm exists or is known that is able to compute the solution to the problem rather quickly (in polynomial time complexity).
+    A problem is considered to be computationally hard if no algorithm exists or is known that is able to compute the solution to the problem relatively quickly (in polynomial time complexity).
 
 confirmations::
     Once a transaction is included in a block, it has one confirmation. As soon as _another_ block is mined on the same blockchain, the transaction has two confirmations, and so on. Six or more confirmations are considered sufficient proof that a transaction cannot be reversed.
 
 contract::
-    A contract is a set of Bitcoin transactions which result together in a certain desired behavior.
+    A contract is a set of Bitcoin transactions which together result in a certain desired behavior.
     Examples are RSMCs to create a trustless, bi-directional payment channel or HTLCs to create a mechanism which allows trustless forwarding of payments through third parties.
 
-Diffie Hellman Key Exchange::
+Diffie Hellman (DH) Key Exchange::
     On the Lightning Network, the Elliptic Curve Diffie-Hellman method is used.
     It is an anonymous key agreement protocol that allows two parties, each having an elliptic-curve public-private key pair, to establish a shared secret over an insecure communication channel.
     This shared secret may be directly used as a key, or to derive another key.
     The key, or the derived key, can then be used to encrypt subsequent communications using a symmetric-key cipher.
-    An example of the derived key would be the shared secret between the ephemeral session key of a sender of an onion with the nodes public key of a hop of the onion as described and used by the SPHINX Mix Format.
-    Via https://en.wikipedia.org/w/index.php?title=Elliptic-curve_Diffie%E2%80%93Hellman&oldid=836070673
-
+   
```

Comment:
I understand the reasoning behind the thought of removing this, however I'm not sure if it's right thing to do. If the person reading doesn't want to dive deep, they would just skip the link. Since it's in the glossary, I don't think it clogs up the book

Reply:
My reasoning: A glossary should define and explain a term and not introduce new complexity.  What is "an onion" and a "hop of the onion"? What is SPHINX Mix Format?  I think it is good to have an example, but it should be understandable without having to find other terms in the Glossary.

Reply:
I agree, the SPHINX Mix Format sentence is kinda hard to understand without context. Should we keep the link to Wikipedia though? Or should we just remove it?

Reply:
I'd remove the link.  It doesn't add authority, and the wording is different.  The readers of the book will know how to do a search!

Reply:
I am a little uncertain. I agree that for a glossary it is too long